develop4learners/SocialJustice-CS-game
# Student notebook refactor

- Added the option for students to write personal notes into their notebook. 
- Made notebook circular (can press back button on first slide to go to last slide)
- Updated images with higher res versions

Testing:
    1. Go to a place in the game where you have access to the notebook.
    2. Press the back button (to ensure that the circular notebook works)
    3. Verify that it shows the correct message, something along the lines of "You have no notes, go to the studentNotebook.txt file"
    4. Close the notebook.
    5. Navigate to the studentNotebook.txt file in the game/ folder
    6. Replace the text there and save the file.
    7. Open the notebook again. Your changes should reflect on the Notes page and should not overflow off notebook.
